packages/liveblocks-react-ui/src/utils/use-transition.ts

```diff
@@ -8,8 +9,13 @@ const useReactTransition: typeof React.useTransition = (React as any)[
   "useTransition".toString()
 ];
 
-function useTransitionFallback(transition: () => void) {
-  return [false, transition] as ReturnType<typeof useReactTransition>;
+function useTransitionFallback() {
+  const startTransition = useCallback(
+    (callback: TransitionFunction) => callback(),
+    []
+  );
+
+  return [false, startTransition] as ReturnType<typeof useReactTransition>;
```

Comment:
Prefer an explicit return type annotation over `as`-casting (which can hide bugs and cause headaches later):

```suggestion
function useTransitionFallback(): ReturnType<typeof useReactTransition> {
  const startTransition = useCallback(
    (callback: TransitionFunction) => callback(),
    []
  );

  return [false, startTransition];
```